sql/pg_duckdb--0.2.0--0.3.0.sql

```diff
@@ -0,0 +1,9 @@
+CREATE FUNCTION @extschema@.approx_count_distinct(a anyelement)
```

Comment:
I'm wondering if this should be created in `@extschema@` (i.e. the `public` schema) or the `duckdb` schema. I think `@extschema@` is probably fine.

Reply:
I don't have a strong opinion. I was just following the existing functions